CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 Here we document changes that affect the public API or changes that needs to be communicated to other developers. 
 
+## 2020-06-03 WebBrowser Javascript API
+Changed redirection of `https://inviwo` to `api://inviwo` to avoid confusion with the https scheme.
+Your html-files will need to update from 
+'https://inviwo/modules/yourmodule' 
+to 
+'api://inviwo/modules/yourmodule' 
```

Comment:
is api recommended practice? why not use inviwo:// instead? 
also not that new modules don't use the modules prefix in the path but rather inviwo or some other name specific t the authors. Hence for new core modules you would get api://inviwo/inviwo/newmodule
if it is possible I think inviwo://mymodule/... would be best? why do you even need the "module" in there? should not the module id be enough?

Reply:
good idea!

Reply:
we have both app/ and modules/, hence the separation

Reply:
would it be possible to just use core / app / module_identifier? Guess we could assume there are no module name app...

Reply:
fixed